app/views/install/compose.phtml

```diff
@@ -345,6 +345,8 @@ services:
       - _APP_DB_PASS
       - _APP_LOGGING_PROVIDER
       - _APP_LOGGING_CONFIG
+      - _APP_WORKERS_NUM=1
+      - _APP_QUEUE_NAME=database_db_main
```

Comment:
please double check the name of the default database. Im not sure its `db_main`

Reply:
it is

Reply:
lets remove these variables from here and hard code them as default values in the code